src/index.js

```diff
@@ -102,6 +102,16 @@ inquirer.prompt(questionnaire).then(async (answers) => {
           }
         }
       );
+
+      fs.copyFile(
+        `${currentPath}/themeTemplates/react-themes/dark-theme.constants.js`,
+        `${frontEnd.path}/src/dark-theme.constants.js`,
+        (err) => {
+          if (err) {
+            console.error("Getting error while copying theme constants file:", err);
```

Comment:
for error message:
Getting error while copying theme constants file:
=>
Error while copying theme's static files